component/identity/src/test/resources/conf/picketlink/exo.portal.component.identity-picketlink-idm-config.xml

```diff
@@ -30,6 +30,24 @@
       <identity-type-mappings>
         <user-mapping>USER</user-mapping>
       </identity-type-mappings>
+      <options>
+        <option>
+          <name>template</name>
+          <value>true</value>
+        </option>
+        <option>
+          <name>credentialEncoder.class</name>
+          <value>org.picketlink.idm.impl.credential.DatabaseReadingSaltEncoder</value>

```

Comment:
Should we keep the old encoder here ?

Reply:
the new one will cause an issue in the integration tests of userDaoImpl,  because the web.security dependencies is not available using the old one will will create a user with the old encoder and then due to the integration test, it will update the user password by the new encoder